bin/libs/component.sh

```diff
@@ -408,6 +408,154 @@ process_zss_plugin_install() {
   fi
 }
 
+process_zis_plugin_install() {
+  if [ "${ZWE_RUN_ON_ZOS}" = "true" ]; then
+    zwes_zis_pluginlib=$(read_yaml "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.dataset.authPluginLib")
+    zwes_zis_parmlib=$(read_yaml "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.dataset.parmlib")
+    zwes_zis_parmlib_member="ZWESIP00"
+    print_trace "- Checking for zis plugins and verifying them"
+    component_dir="${1}"
+    
+    iterator_index=0
+    zis_plugin_id=$(read_component_manifest "${component_dir}" ".zisPlugins[${iterator_index}].id" 2>/dev/null)
+    zis_plugin_path=$(read_component_manifest "${component_dir}" ".zisPlugins[${iterator_index}].path" 2>/dev/null)
+    while [ -n "${zis_plugin_path}" ]; do
+      cd "${component_dir}"
+      zis_plugin_install "${zis_plugin_path}" "${zwes_zis_pluginlib}" "${zwes_zis_parmlib}" "${zwes_zis_parmlib_member}" "${zis_plugin_id}" "${component_dir}"
```

Comment:
The caller should check whether the function fails or not, display error messages, and instruct what the user can do.